app/models/subscription/responses/SubscriptionInfo.scala

```diff
@@ -25,7 +25,13 @@ final case class SubscriptionInfo(id: String,
                                   gbUser: Boolean,
                                   tradingName: Option[String],
                                   primaryContact: Contact,
-                                  secondaryContact: Option[Contact])
+                                  secondaryContact: Option[Contact]) {
+
+  lazy val primaryContactName: String = primaryContact match {
```

Comment:
Is this tested? I can't see test for it.

Reply:
SubscriptionInfo tests added.